raft-tech/cfgov-refresh
# Mm/change add input screen details

Client wished to change the labeling on the add income/add expense screens.  This is issue #203 in cfgov/money-tools project board.

## Removals

- Removed the word 'Pay' and 'Due' from labeling on form.js.

## How to test this PR

1. Open app and clear app cache.
2. Move through app to get to calendar.  There is no need to add a _Starting Balance_.
3.  Add an expense.  Note the format of the screen.  It should look like the expense screenshot below.  
4.  Add an income.  Note the format of the screen.  It should look like the income screenshot below.

## Screenshots
<img width="150" alt="Screen Shot 2020-08-12 at 4 36 32 PM" src="https://user-images.githubusercontent.com/34319929/90066759-2de27680-dcbc-11ea-9ffe-88804ddbac16.png">
<img width="150" alt="Screen Shot 2020-08-12 at 4 36 46 PM" src="https://user-images.githubusercontent.com/34319929/90066761-2de27680-dcbc-11ea-8440-600c47b9c341.png">
